docs/DeveloperGuide.adoc

```diff
@@ -246,39 +246,54 @@ image::DeletePersonSdForLogic.png[width="800"]
 [[Design-Model]]
 === Model component
 
-The `Model` component consist of a `ModelManager` that manages all the models.
+The `Model` component is managed by the `ModelManager` that stores the data of Car Park Finder.
+It does not depend on any of the three other components.
+
+Please refer to the Class Diagram below for more details.
 
 .Structure of the Model Component
-image::ModelClassDiagram.png[width="800"]
 
+image::ModelClassDiagram.png[width="800"]
 *API* : link:{repoURL}/src/main/java/seedu/address/model/Model.java[`Model.java`]
 
-The `Model`,
 
-* stores a `UserPref` object that represents the user's preferences.
-* stores the Car Park Finder data.
-* exposes an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
-* does not depend on any of the other three components.
+In general, this is the structure of the Model Component:
+
+* The `ModelManager` extends the `Model` Interface.
+
+* It stores a `VersionedAddressBook` and a `UserPref` object.
+** The `UserPref` object represents the user's preferences.
+** The `VersionedAddressBook` contains a `addressBookStateList` which is used to store multiple `Carpark` objects.
+Each `Carpark` object has various attributes; `address`, `carparkNumber`, `coordinate`, `carparkType`, `freeParking`,
+`lotsAvailable`, `nightParking`, `shortTerm`, `totalLots`, `typeOfParking`,
+`tags`
+
+* The `ModelManager` also manages a filtered list of `Person(?) Carpark` objects filtered from the `UniquePersonList`.
+
+* The Model component exposes an unmodifiable `ObservableList<Carpark>` that can be 'observed'
+e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list changes.
+
 
 [NOTE]
 As a more OOP model, we can store a `Tag` list in `Car Park Finder`, which `Carpark` can reference. This would allow `Car Park Finder` to only require one `Tag` object per unique `Tag`, instead of each `Carpark` needing their own `Tag` object. An example of how such a model may look like is given below. +
  +
 image:ModelClassBetterOopDiagram.png[width="800"]
+Use of Tag object
```

Comment:
I can't really see the doc properly but why is there a stray text here?

Reply:
Oh this one is supposed to be a caption to the diagram, but for some reason the font and colour isnt the same as the other captions although the same notation is used

Reply:
If its a caption then follow the style of the rest. Use a full stop and place the caption above the picture.